docs/docs/projects/index.mdx

```diff
@@ -2,6 +2,634 @@
 sidebar_position: 13
 ---
 
+import { APILink } from "@site/src/components/APILink";
+import TOCInline from "@theme/TOCInline";
+
 # MLflow Projects
 
-{/* WIP */}
+An MLflow Project is a format for packaging data science code in a reusable and reproducible way,
+based primarily on conventions. In addition, the Projects component includes an API and command-line
+tools for running projects, making it possible to chain together projects into workflows.
+
+<TOCInline toc={toc} maxHeadingLevel={2} minHeadingLevel={2} />
+
+## Overview
+
+At the core, MLflow Projects are just a convention for organizing and describing your code to let
+other data scientists (or automated tools) run it. Each project is simply a directory of files, or
+a Git repository, containing your code. MLflow can run some projects based on a convention for
+placing files in this directory (for example, a `conda.yaml` file is treated as a
+[Conda](https://conda.io/docs) environment), but you can describe your project in more detail by
+adding a `MLproject` file, which is a [YAML](https://learnxinyminutes.com/docs/yaml) formatted
+text file. Each project can specify several properties:
+
+<dl>
+  <dt>Name</dt>
```

Comment:
Could we make the `<dt>` contents subheaders also? It might be nice to visually distinguish them from the text, and also then the contents would appear in the sidebar

```suggestion
  <dt>### Name</dt>
```

Reply:
@daniellok-db Done. I have also done it with the `dt`s in `model-registry` for consistency.